app.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+import aws_cdk as cdk
+import recipes_dlz as dlz
+from data_landing_zone_example_python.minimum_config import config
+
+app = cdk.App()
+
+dlz.DataLandingZone(
+    app,
+    local_profile=config.local_profile,
+    budgets=config.budgets,
+    mandatory_tags=config.mandatory_tags,
+    organization=dlz.DLzOrganization(
```

Comment:
In the complete config, you don't use these classes but only object notation. I think pick one and stick with it, maybe the object notation is better/less explicit but it reads better for sure.